src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/LicenceController.kt

```diff
@@ -713,7 +713,7 @@ class LicenceController(
   fun activateLicences(
     @Valid @RequestBody request: List<Long>
   ) {
-    licenceService.activateLicences(request)
+    licenceService.activateLicencesByIds(request)
```

Comment:
are we going to be able to get rid of these after moving over to the new endpoint?

Reply:
if so you could add a @Deprecated annotation to methods that can be deleted. This will grey them out of swagger docs and add warnings in intelliJ if you use them

Reply:
We may be able to get rid of the activation method, but there's a different process that hits the deactivation endpoint (the licence expiry job for when licences reach their end dates).
Truthfully I was leaving it for now until the updated job is working, just in case something goes wrong, because then all we have to do is rollback the front-end changes and the old job will still work.

Reply:
That's fine - good to add deprecated if it's your intention to remove the active one. Rollback would revert the annotation but it signifies to people "do not use it" and "this will go"